lutris/game_actions.py

```diff
@@ -87,14 +93,18 @@ def on_locate_installed_game(self, *_args):
 
     @property
     def is_game_removable(self):
-        return self.game and (self.game.is_installed or self.game.is_db_stored)
+        for game in self.games:
+            if not (game and (game.is_installed or game.is_db_stored)):
+                return False
+        return True
 
     def on_remove_game(self, *_args):
         """Callback that present the uninstall dialog to the user"""
-        if self.game.is_installed:
-            UninstallGameDialog(game_id=self.game.id, parent=self.window).run()
-        else:
-            RemoveGameDialog(game_id=self.game.id, parent=self.window).run()
+        for game in self.games:
+            if game.is_installed:
+                UninstallGameDialog(game_id=game.id, parent=self.window).run()
+            else:
+                RemoveGameDialog(game_id=game.id, parent=self.window).run()
 
     def on_view_game(self, _widget):
         """Callback to open a game on lutris.net"""
```

Comment:
You missed this one; this should be a loop too.

Reply:
Did not understand. Uninstalling and removing is already in a loop.

Reply:
on_view_game() is not a loop, that's what I meant.

Reply:
Yeah. Did not make all of them loops. Only worked on the code to uninstall/remove. Will make them all loops then.

Reply:
Hmm. I didn't mean that the MultipleGameActions class should go; just loops in the base class. But this resolution is okay; multi-select support for more commands should be easier this way.